### ISSUE TITLE
Postgres issue

### ISSUE THREAD

**Issue by KatieMFritz** [flipboxfactory/organizations#13]
Setting up organization fields, org types, and user types worked okay, but when I try to create a new organization, I get this message, which I believe is a syntax issue affecting PosgreSQL.

```
Database Exception – yii\db\Exception
SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for integer: "f"
The SQL being executed was: SELECT "organizations_types".*
FROM "organizations_types" "organizations_types"
LEFT JOIN "organizations_types_associations" "organizations_types_associations" ON "organizations_types_associations"."typeId" = "organizations_types"."id"
WHERE "organizations_types_associations"."organizationId"=FALSE
LIMIT 1
Error Info: Array
(
    [0] => 22P02
    [1] => 7
    [2] => ERROR:  invalid input syntax for integer: "f"
)
↵
Caused by: PDOException
SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for integer: "f"
in /home/forge/dev.facadetectonics.org/vendor/yiisoft/yii2/db/Command.php at line 1290
```

![Screen Shot 2019-04-29 at 9 35 49 PM](https://user-images.githubusercontent.com/7085184/56936699-d7e9bf80-6ac6-11e9-98eb-62dd3e91bc0a.png)

**Comment by nateiler** [comment]
Looks like a Postgres specific error to me.

I've got some updates for you:

`composer require flipboxfactory/organizations:^1.0.8`

There is a supporting package that should also get updated (as it's required).  However if three is a dependency notice, you might need to run this as well: `composer require flipboxfactory/craft-ember:^2.3.1`

**Comment by KatieMFritz** [comment]
Thanks! That problem is solved, but something I'm noticing in this and also asset list fields is that relations are getting duplicated or triplicated.

Here's a video showing how adding a person using the User List field type (part of Organizations, right?) adds the person 3 times.

https://cl.ly/21c962ac1e89

With the Asset List, I noticed that if I added one of my images, it would add it twice. Deleting and re-uploading the image fixed it. I don't want to do that with users though - these are actual users.

**Comment by KatieMFritz** [comment]
☝🏻I think that issue is actually for element lists, not for this plugin.

**Comment by nateiler** [comment]
Yeah, it is.

Try release v2.1.1.1 

Technically, multiple relations were not getting created, but the relational query join was making them appear as if they were.  That bug was introduced in v2.1.1 and fixed in v2.1.1.1
